en/index.md

```diff
@@ -523,6 +523,26 @@ Lastly, these Latin words and phrases are widespread in academia and research. T
 - stet
 
 
+<!-- RULE
+#26 Avoid orphan demonstrative pronouns
+-->
+
+## Lone "this", "these", "those", "that"
+
+A _demonstrative pronoun_ is a word that refers to a thing previously discussed, and are the familiar words "this", "these", "those" and "that". They can work both in isolation ("That's great!") and paired with the thing they refer to ("This apple is sweet.").
```

Comment:
Is the "demonstrative pronoun **is**..." followed by "and **are**..." a little awkward here?

Maybe:

> ...demonstrative pronouns are words that... The familiar words... are all demonstrative pronouns.

or

>A demonstrative pronoun is... The familiar words... are all demonstrative pronouns

Reply:
Oops, bad grammar, good catch. Me write good. 06c9bb481a117e9b09a6e12deb8123dfc94040c2